tests/cli/test_dataset_config_generation.py

```diff
@@ -0,0 +1,135 @@
+from dsgrid.dimension.base_models import DimensionType
+
+from click.testing import CliRunner
+
+from dsgrid.cli.dsgrid import cli as cli
+from dsgrid.tests.common import TEST_PROJECT_PATH
+from dsgrid.utils.files import load_data
+
+
+def test_generate_dataset_config_pivoted_matches(cached_registry, tmp_path):
+    conn = cached_registry
+    runner = CliRunner(mix_stderr=False)
+    dataset_id = "comstock"
+    output_dir = tmp_path / dataset_id
+    assert not output_dir.exists()
+    cmd = [
+        "--url",
+        conn.url,
+        "registry",
+        "datasets",
+        "generate-config",
+        dataset_id,
+        f"dsgrid-test-data/datasets/test_efs_{dataset_id}",
+        "--project-id",
+        "test_efs",
+        "--schema-type",
+        "standard",
+        "--pivoted-dimension-type",
+        "metric",
+        "--time-columns",
+        "timestamp",
```

Comment:
This is the default, so could (also) test that the command works without this option.

Reply:
The next dataset does that.